deployment/refinery-modules/refinery/manifests/python.pp

```diff
@@ -1,11 +1,10 @@
 class refinery::python {
 
   class { '::python':
-    version    => 'system',
-    pip        => true,
-    dev        => true,
-    virtualenv => true,
-    gunicorn   => false,
+    dev        => 'present',
```

Comment:
Yes but even though other params have defaults too, I still had to assign values explicitly for some reason.